client/my-sites/domains/domain-management/list/domain-only.jsx

```diff
@@ -56,11 +56,11 @@ const DomainOnly = ( { primaryDomain, hasNotice, recordTracks, siteId, slug, tra
 			<EmptyContent
 				title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
 				line={ translate( 'Start a site now to unlock everything WordPress.com can offer.' ) }
-				action={ translate( 'Create Site' ) }
+				action={ translate( 'Create site' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.